spirv_glsl.cpp

```diff
@@ -9606,6 +9606,9 @@ string CompilerGLSL::access_chain_internal(uint32_t base, const uint32_t *indice
 	bool pending_array_enclose = false;
 	bool dimension_flatten = false;
 
+	// If we are translating access to a structured buffer, the first subscript '._m0' must be hidden
+	const bool hide_first_subscript = count > 1 && is_user_type_structured(base) ? 1 : 0;
```

Comment:
Need to verify that back to back AccessChain works. E.g. AccessChain into an array of SSBOs, then access chain into a particular member, or access chain down to a vector, then access chain down to a scalar for example.